src/Parser/PhpParser.php

```diff
@@ -75,10 +75,11 @@ private function getFileContent($filename, $lineNumber)
             return null;
         }
 
-        $content = '';
         $lineCnt = 0;
-        $file = new SplFileObject($filename);
-        while (!$file->eof()) {
+        $content = '';
+        $file    = new SplFileObject($filename);
```

Comment:
I suggest not aligning ``=`` signs. This creates useless git conflicts when adding new variables, and makes ``git blame`` much less useful by filling the history with meaningless changes on lines unrelated to the actual change of the commit